src/applications/vaos/appointment-list/pages/UpcomingAppointmentsDetailsPage/VideoLink.unit.spec.jsx

```diff
@@ -30,13 +38,13 @@ describe('VAOS Component: VideoLink', () => {
       .be.true;
   });
   it('renders disabled join appoinment link 35 minutes prior to start time', () => {
-    const now = moment();
+    const now = zonedTimeToUtc(subMinutes(new Date(), 35), 'America/New_York');
     const appointment = {
       location: {
         vistaId: '983',
         locationId: '983',
       },
-      start: moment.tz(now, 'America/New_York').subtract(35, 'minutes'),
+      start: formatInTimeZone(now, 'America/New_York', "yyyy-MM-dd'T'HH:mm:ss"),
```

Comment:
FYI, I've been adding format strings [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/constants.js#L496) to reduce the number of magic strings everywhere. What do you think?

Reply:
I will update tests to use.